cmake/packaging.cmake

```diff
@@ -40,9 +40,12 @@ if(WIN32 AND NOT UNIX)
    !include \\\"${PATH_TO_PLUGINS}\\\\FileAssociation.nsh\\\"")
 
   # Create association on install
+  set(F3D_REGISTER_LIST "${F3D_FILE_ASSOCIATION_NSIS}")
+  list(TRANSFORM F3D_REGISTER_LIST PREPEND "\\\${RegisterExtension} '\\\"$INSTDIR\\\\bin\\\\f3d.exe\\\"' ")
+  list(JOIN F3D_REGISTER_LIST "\n      " F3D_REGISTER_STRING)
   set(CPACK_NSIS_EXTRA_INSTALL_COMMANDS "\
     StrCmp $REGISTER_EXTENSIONS \\\"0\\\" doNotRegisterExtensions\n\
-      !include \\\"NSIS.RegisterCommands.nsh\\\"\n\
```

Comment:
This include should be somewhere, but I cant find it anywhere.

Reply:
unless it is actually not needed.

Reply:
It's not needed anymore